src/components/Kv/KvCodeBlock.vue

```diff
@@ -0,0 +1,39 @@
+<template>
+	<div class="code" :class="`${nowrap ? 'nowrap' : ''}`">
+		<pre v-if="code">{{ code }}</pre>
+		<slot v-else></slot>
+	</div>
+</template>
+
+<script>
+export default {
+	props: {
+		code: {
+			type: String,
+			default: '',
+		},
+		nowrap: {
+			type: Boolean,
+			default: false,
+		},
+	},
+};
+</script>
+
+<style lang="scss" scoped>
+@import 'settings';
+
+.code {
```

Comment:
please move these to typography.scss globals

Reply:
Cool, done 👍